Makefile

```diff
@@ -55,11 +63,11 @@ version:
 
 operator: codegen
 	mkdir -p $(BIN)
-	go build $(LDFLAGS) -o $(BIN)/operator cmd/operator/main.go
+	go build $(BUILD_OPTIONS) -o $(BIN)/operator cmd/operator/main.go
 
 talisman: codegen
 	mkdir -p $(BIN)
-	go build $(LDFLAGS) -o $(BIN)/talisman cmd/talisman/talisman.go
```

Comment:
Please double-check if this really produces static binaries (i.e. `ldd executable`).
The reason why I'm asking, is because it looks like the "GOENV" has been currently used only as a Makefile-variable -- if you want to use it in the build, you'd need something like `env $(GOENV) $(GO) build $(BUILD_OPTIONS) ...`  (see https://github.com/portworx/px-installer/blob/master/Makefile#L64)

Reply:
Updated the makefile and verified it works.